src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -299,6 +299,10 @@ class TableToolPage extends Component<Props, State> {
       this.createQuery(filters, indicators),
     );
 
+    if (result.length < 1) {
+      throw new Error('There is no data for the options selected');
```

Comment:
Throwing an error seems like a bad idea as it will probably throw the user to the error page. This is a **really** bad user experience as they then have to go back and re-input all of their choices.

We should be failing gracefully and just displaying an error message where the table would normally be rendered.